lager/lenses.hpp

```diff
@@ -123,51 +128,103 @@ auto attr(Member member)
     });
 }
 
+/*!
+ * `Key -> Lens<{X}, [X]>`
+ */
```

Comment:
The `at(key)` family of lenses now returns an optional instead of a default constructed value if the target is not present.